core/module.go

```diff
@@ -66,6 +66,32 @@ type Module struct {
 	InstanceID *call.ID
 }
 
+type SDKConfig struct {
+	Source string            `field:"true" name:"source" doc:"Source of the SDK. Either a name of a builtin SDK or a module source ref string pointing to the SDK's implementation."`
```

Comment:
:thinking: One day, I wonder if we would want to split this up into a `source` and `builtin` field. But that's kind of out-of-scope for here, just wondering out loud.